pkg/interface/src/views/apps/notifications/notification.tsx

```diff
@@ -100,7 +100,7 @@ function NotificationWrapper(props: {
         </StatelessAsyncAction>
         {!props.archived && (
           <StatelessAsyncAction name={time.toString()} onClick={onArchive} backgroundColor="transparent">
-            Archive
+            Read
```

Comment:
This might be a bit confusing, since the blue dot also indicates "read" state, but is separate from this. "Dismiss" might be more appropriate?

Reply:
Oh yes, I should have commented here — let's definitely go with dismiss.

I mentioned this in the comparison screens I had sent Liam, which I can repost here:

Before:
<img width="1278" alt="2020 12 17 00 53 00-Screen Shot 2020-12-16 at 7,52,37 PM" src="https://user-images.githubusercontent.com/1195363/102656203-7fef4900-4141-11eb-83f1-9ab22ae6aa1e.png">

After:

<img width="1258" alt="2020 12 17 00 53 05-Screen Shot 2020-12-16 at 7,52,52 PM" src="https://user-images.githubusercontent.com/1195363/102656228-87aeed80-4141-11eb-9d14-9e3a8fa0de39.png">

Reply:
I can change the name to Dismiss. We should clean up the attributes in the store / etc when the archival functionality is removed from `%hark-store` itself, which is slated to occur early in Q1.